### ISSUE TITLE
Docker: labgrid-exporter needs adjustement for /var/cache/labgrid

### ISSUE THREAD

**Issue by Emantor** [labgrid-project/labgrid#393]
Labgrid now stores managedfiles inside /var/cache/labgrid. The container Dockerfile should probably create the necessary directory or use the example tmpfiles configuration file. cc/ @esben

**Comment by esben** [comment]
Would it make sense to expose it as a volume, so users can have it persist between containers?

**Comment by Emantor** [comment]
Yep, thats a good idea. The user should also be notified that the volume needs to be cleaned up from time to time.

**Comment by jluebbe** [comment]
How often do you expect to restart the container? For the systemd-tempfiles entry we recommend two days, as we don't think the images are useful that long in most cases.
